### DISCUSSION TITLE
Any plans for adding any console emulators support?

### DISCUSSION THREAD

**Discussion by Bricciooo** [Heroic-Games-Launcher/HeroicGamesLauncher#3995] [Ideas]
I would really like to play some PS2 and Nintendo DS titles within the launcher (just like in lutris with their runners), and I think it would just be more options to have your games in one place. :)
![image](https://github.com/user-attachments/assets/50f486b8-1794-4adf-9f08-9619574c42ea)

**Comment by vlad-cool** [comment]
I think external emulators can be added to heroic as a Linux game. Maybe some install script would be convenient enough

**Comment by vlad-cool** [comment]
Of corse native option would be much better
